ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/KsqlErrorMessage.java

```diff
@@ -30,23 +30,26 @@
 import java.util.stream.Collectors;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
-@JsonTypeInfo(
-    use = JsonTypeInfo.Id.NAME,
-    include = JsonTypeInfo.As.PROPERTY)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME)
 @JsonSubTypes({
     @JsonSubTypes.Type(value = KsqlErrorMessage.class, name = "generic_error"),
-    @JsonSubTypes.Type(value = KsqlStatementErrorMessage.class, name = "statement_error")})
-public class KsqlErrorMessage extends ErrorMessage {
+    @JsonSubTypes.Type(value = KsqlStatementErrorMessage.class, name = "statement_error")
+})
+public class KsqlErrorMessage {
```

Comment:
Removed the dependency on `ErrorMessage`, which is a rest-util type and hence we shouldn't be exposing.